Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1013,6 +1015,10 @@ class RunDashboardTests(RunWebKitTests):
     descriptionDone = ["dashboard-tests"]
     resultDirectory = os.path.join(RunWebKitTests.resultDirectory, "dashboard-layout-test-results")
 
+    def __init__(self, *args, **kwargs):
+        kwargs['report'] = kwargs.get('report', False)
```

Comment:
using `kwargs` in this manner seems pretty weird/confusing. Can simply use a class variable (for e.g. see `STRESS_MODE` in https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/steps.py#L4393 which achieves similar functionality through class variable).